### PR TITLE
fix: apply dropdown-button slot and initial min-width

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

image

### CODE REVIEW

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 1]
```diff
@@ -355,7 +355,7 @@ watch(() => props.disabled, (disabled) => {
         width: 100%;
     }
 
-    &.icon-button {
+    &.icon-button, &.transparent {
```
Comment (WANZARGEN) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:358]: praise: 👍

File: packages/mirinae/src/inputs/forms/json-schema-form/PJsonSchemaForm.vue [hunk 2]
```diff
@@ -82,7 +82,7 @@
                                                class="input-form select-dropdown"
                                                @update:selected="handleUpdateFormValue(schemaProperty, propertyIdx, ...arguments)"
                             >
-                                <template #default="{ item }">
+                                <template #dropdown-button="item">
```
Comment (WANZARGEN) [inline packages/mirinae/src/inputs/forms/json-schema-form/PJsonSchemaForm.vue:85]: praise: Thank you!

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Review by WANZARGEN — COMMENTED** [review]
image

Oh, there is some issue!
The dropdown arrow must be stick into the right side.
This issue is on DashboardWidgetInputForm that customizes json schema form.

**Review by yuda110 — APPROVED** [review]
LGTM!!!

**Pull request merged**